index.css

```diff
@@ -0,0 +1,108 @@
+html body {
+  margin: 0;
+  padding: 0;
+}
+
+.main-container {
+  display: flex;
+  flex-direction: column;
+}
+
+.background-1 {
+  position: absolute;
+  right: 0;
+  height: 256px;
+  top: 430px;
+}
+
+.top-nav {
+  display: flex;
+  background: #3c3a39;
+  height: 70px;
+  align-items: center;
+  justify-content: space-between;
+}
+
+.link-name {
+  color: white;
+  text-decoration: none;
+  margin-left: 24px;
+  margin-top: 20px;
+  height: 24px;
+  font-size: large;
+  font-style: normal;
+  font-family: 'Roboto', sans-serif;
+}
+
+.hamburger .bar {
+  display: block;
+  width: 30px;
+  height: 5px;
+  background-color: white;
+  margin-top: 5px;
+  margin-right: 24px;
+  border-radius: 10px;
+}
+
+.intro {
+  display: flex;
+  padding: 0;
+  margin: 0;
+  flex-direction: column;
+  background-color: black;
+  height: 600px;
+  align-items: flex-start;
+}
```

Comment:
- [ ] [Optional] Kindly be noticed that you can make the height of the div as the height of any screen by using 100vh but you need to subtract from it the height of the nav bar.
```css
.intro {
  display: flex;
  padding: 0;
  margin: 0;
  flex-direction: column;
  background-color: black;
  min-height: calc(100vh - 70px);
  align-items: flex-start;
}
```